anchor.js

```diff
@@ -134,11 +134,11 @@
         readableID = elementID.replace(/-/g, ' ');
 
         // The following code builds the following DOM structure in a more effiecient (albeit opaque) way.
-        // '<a class="anchorjs-link ' + this.options.class + '" href="#' + elementID + '" aria-label="Anchor link for: ' + readableID + '" data-anchorjs-icon="' + this.options.icon + '"></a>';
+        // '<a class="anchorjs-link ' + this.options.class + '" href="#' + elementID + '" aria-label="Anchor" data-anchorjs-icon="' + this.options.icon + '"></a>';
         anchor = document.createElement('a');
         anchor.className = 'anchorjs-link ' + this.options.class;
         anchor.href = '#' + elementID;
-        anchor.setAttribute('aria-label', 'Anchor link for: ' + readableID);
+        anchor.setAttribute('aria-label', 'Anchor');
```

Comment:
I would suggest the aria-label text be configurable via an option so that it could support multiple languages.

Reply:
Good idea. Let me see what I can do here.

Reply:
if you had `options.ariaLabel`, then you could test to see if it is `truthy`, and if it is, then set the `aria-label` attribute, otherwise don't add it at all.

```js
if (this.options.ariaLabel) {
  anchor.setAttribute('aria-label',this.options.ariaLabel);
}
```